diff --unified a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2016-08-13 00:32:09.362915666 +0100
+++ b/PKGBUILD	2016-08-13 00:31:35.306249247 +0100
@@ -2,8 +2,8 @@
 # Maintainer: Tobias Powalowski <tpowa@archlinux.org>
 # Maintainer: Thomas Baechler <thomas@archlinux.org>
 
-pkgbase=linux               # Build stock -ARCH kernel
-#pkgbase=linux-custom       # Build kernel with a different name
+#pkgbase=linux               # Build stock -ARCH kernel
+pkgbase=linux-icarus       # Build kernel with a different name
 _srcname=linux-4.7
 pkgver=4.7
 pkgrel=1
@@ -20,7 +20,8 @@
         'config' 'config.x86_64'
         # standard config files for mkinitcpio ramdisk
         'linux.preset'
-        'change-default-console-loglevel.patch')
+        'change-default-console-loglevel.patch'
+        'acs_override.patch')
 
 sha256sums=('5190c3d1209aeda04168145bf50569dc0984f80467159b1dc50ad731e3285f10'
             'SKIP'
@@ -49,6 +50,9 @@
   # (relevant patch sent upstream: https://lkml.org/lkml/2011/7/26/227)
   patch -p1 -i "${srcdir}/change-default-console-loglevel.patch"
 
+  # ICARUS: Apply ACS Override Patch
+  patch -p1 -i "${srcdir}/acs_override.patch"
+
   if [ "${CARCH}" = "x86_64" ]; then
     cat "${srcdir}/config.x86_64" > ./.config
   else
