diff -u a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2016-09-19 09:44:36.132996762 +0100
+++ b/PKGBUILD	2016-09-19 09:48:47.199655969 +0100
@@ -2,8 +2,8 @@
 # Maintainer: Tobias Powalowski <tpowa@archlinux.org>
 # Maintainer: Thomas Baechler <thomas@archlinux.org>
 
-pkgbase=linux               # Build stock -ARCH kernel
-#pkgbase=linux-custom       # Build kernel with a different name
+#pkgbase=linux              # Build stock -ARCH kernel
+pkgbase=linux-icarus        # Build kernel with a different name
 _srcname=linux-4.7
 pkgver=4.7.4
 pkgrel=1
@@ -21,6 +21,7 @@
         # standard config files for mkinitcpio ramdisk
         'linux.preset'
         'change-default-console-loglevel.patch'
+        'acs_override.patch'
         )
 
 sha256sums=('5190c3d1209aeda04168145bf50569dc0984f80467159b1dc50ad731e3285f10'
@@ -52,6 +53,9 @@
   # (relevant patch sent upstream: https://lkml.org/lkml/2011/7/26/227)
   patch -p1 -i "${srcdir}/change-default-console-loglevel.patch"
 
+  # ICARUS: Apply ACS override patch
+  patch -p1 -i "${srcdir}/acs_override.patch"
+
   if [ "${CARCH}" = "x86_64" ]; then
     cat "${srcdir}/config.x86_64" > ./.config
   else
